pkg/provider/traefik/internal_test.go

```diff
@@ -205,7 +205,7 @@ func Test_createConfiguration(t *testing.T) {
 						},
 					},
 					"websecure": {
-						Address: ":443",
```

Comment:
@SantoDE Adding a specific test case for the trailing protocol in the address could be a more explicit. WDYT ?

Reply:
What ever you prefer to be honest. I can live with both.

Reply:
Ok 👍i'll do it